packages/aws-rfdk/docs/upgrade/upgrading-1.5.md.yaml

```
@@ -0,0 +1,13 @@
+# Upgrading to RFDK v1.5.x or Newer
+
+## Updating DocumentDB
+
+Please backup your existing database before upgrading to RFDK 1.5.x.
+
+The default DocumentDB engine version used by RFDK versions >= 1.5.x have been upgraded to 5.0.0. Note that if you have an existing DocumentDB cluster with previous versions, you will need to manually upgrade by following the documentation: https://docs.aws.amazon.com/documentdb/latest/developerguide/docdb-mvu.html
+
+## Updating MongoDB
+
+Please backup your existing database before upgrading to RFDK 1.5.x.
+
+The default MongoDB version used by RFDK versions >= 1.5.x have been upgraded to 8.0. Note that deployments with an existing MongoDB versions lower than 8.0 will not upgrade to the new version. See the following blog post for information on how to manually update your MongoDB database: https://awsthinkbox.zendesk.com/hc/en-us/articles/1500008016222-How-Do-I-Upgrade-MongoDB-to-version-4-2-from-3-2
```

```suggestion
In RFDK versions >= 1.5.x, we upgraded the default MongoDB to 8.0. Note that deployments with an existing MongoDB version lower than 8.0 will not automatically upgrade to the new version. See the following blog post for information on how to manually update your MongoDB database: https://awsthinkbox.zendesk.com/hc/en-us/articles/1500008016222-How-Do-I-Upgrade-MongoDB-to-version-4-2-from-3-2
```

Some nitpicks, and consider changing to active voice.